daprdocs/content/en/developing-applications/building-blocks/actors/howto-actors.md

```diff
@@ -129,3 +129,16 @@ DELETE http://localhost:3500/v1.0/actors/<actorType>/<actorId>/reminders/<name>
 ```
 
 Refer [api spec]({{< ref "actors_api.md#invoke-reminder" >}}) for more details.
+
+## Actor Runtime Configuration
+
+You can configure the Dapr Actors runtime configuration to modify the default runtime behavior.
+
+### Configuration parameters
+- `actorIdleTimeout` - The timeout before deactivating an idle actor. Checks for timeouts occur every `actorScanInterval` interval. **Default: 60 minutes**
+- `actorScanInterval` - The duration which specifies how often to scan for actors to deactivate idle actors. Actors that have been idle longer than actor_idle_timeout will be deactivated. **Default: 30 seconds**
+- `drainOngoingCallTimeout` - The duration when in the process of draining rebalanced actors. This specifies the timeout for the current active actor method to finish. If there is no current actor method call, this is ignored. **Default: 60 seconds**
+- `drainRebalancedActors` - If true, Dapr will wait for `drainOngoingCallTimeout` duration to allow a current actor call to complete before trying to deactivate an actor. **Default: true**
+- `reentrancy` (ActorReentrancyConfig) - Configure the reentrancy behavior for an actor. If not provided, reentrancy is diabled. **Default: disabled**
```

Comment:
@msfussell @berndverst Call this out as a preview feature?

Reply:
Comments
1) Do not call this out as a preview feature since then we care chasing updating docs everywhere. Preview features should be just listed in one place
2) Re this cpmments https://github.com/dapr/docs/issues/1470 the point of this article was to say how to set each of these settings globally, especially default garbage collection. These should be able to be set via the API. If these cannot then that is an issue.

Reply:
@msfussell I missed the `how` in the subject as it was inconsistent with the more specific description. The description of the #1470 is merely saying the defaults aren't documented.

So do we want to document every possible option for these settings here? And do we want to show both SDK and raw REST API?

What is the minimum that is sufficient here? This can easily become a very lengthy section.

Reply:
Yes. Looking at this more, the runtime settings are covered in the SDKs and the topic points to them. So we are good.